modules/CmpdReg/src/client/DeleteLot.coffee

```diff
@@ -0,0 +1,229 @@
+class DeleteLotController extends Backbone.View
+	template: _.template($("#DeleteLotView").html())
+
+	events:
+		"click .cancelDeleteLotButton": "handleCancelButtonClicked"
+		"click .deleteLotButton": "handleDeleteButtonClicked"
+		"click .downloadLotButton": "downloadLot"
+		"click .bv_backToCreg": "handleBackToCregButtonClicked"
+
+
+	initialize: ->
+		_.bindAll(@, 'handleCancelButtonClicked', 'handleDeleteButtonClicked', 'checkDependencies', 'dependencyCheckReturn', 'dependencyCheckError', 'deleteLotError', 'deleteLotReturn', 'downloadLot', 'handleBackToCregButtonClicked');
```

Comment:
Mentioned this in a comment above but yes you are right:

> This is necessary because we are using older backbone which didn't bind @ to the view attributes.